### ISSUE TITLE
to_nextpoi_kqt.py

### ISSUE THREAD

**Issue by Kyraaaaaaaa** [neolifer/LLM4POI#13]
![image](https://github.com/user-attachments/assets/096e4ab2-498b-475a-a5f1-a5c4626758dc)
Is it here 'traj_id' should be str(traj_id)?

**Comment by neolifer** [comment]
Yes. Thanks for pointing out the bug. And I've updated the traj_sim.py with the correct logic along with the files for NYC. You should be able to get the result with them.

**Comment by Kyraaaaaaaa** [comment]
I still cannot get your answer with your updated traj_sim.py file, can you provide your results for traj_sim.py for all three datasets?

**Comment by neolifer** [comment]
Hi there,
The v2 release should solve this problem.
